extensions/patterns/configure_ec2/template_rhdh/create_ec2_instance.yml

```diff
@@ -0,0 +1,185 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: cloud-configure-ec2
+  title: AWS Operations / Create EC2 Instance
+  description: This wizard will guide you on how to create EC2 instance in the Amazon cloud
+  namespace: default
+  tags:
+    - aap-operations
+    - intermediate
+    - cloud-aws-ops
+    - configure-ec2-pattern
+    - create-ec2-instance
+spec:
+  type: service
+  parameters:
+    - title: Prompts
+      description: Create EC2 instance in the Amazon cloud
+      required:
+        - token
+        - inventory
+      properties:
+        token:
+          title: Token
+          type: string
+          description: Oauth2 token
+          ui:field: AAPTokenField
+          ui:widget: password
+          ui:backstage:
+            review:
+              show: false
+        inventory:
+          title: Inventory
+          description: Please enter the inventory you want to use the services on
+          resource: inventories
+          ui:field: AAPResourcePicker
+        credentials:
+          title: Credentials
+          description: Select credentials for accessing the nodes this job will be ran
+            against. You can only select one credential of each type. For
+            machine credentials (SSH), checking "Prompt on launch" without
+            selecting credentials will require you to select a machine
+            credential at run time. If you select credentials and check "Prompt
+            on launch", the selected credential(s) become the defaults that can
+            be updated at run time.
+          type: array
+          ui:field: AAPResourcePicker
+          resource: credentials
+        verbosity:
+          title: Verbosity
+          description: Control the level of output Ansible will produce as the playbook executes.
+          resource: verbosity
+          ui:field: AAPResourcePicker
+    - title: Survey
+      required:
+        - aws_region
+        - instance_name
+        - instance_type
+      description: Create EC2 instance in the Amazon cloud
+      properties:
+        aws_region:
+          title: AWS Region
+          description: AWS region where resources should be created
+          type: string
+          ui:options:
+            rows: 5
```

Comment:
I'm not super familiar with how the backstage templates work, what does `rows: 5` do?